src/python/pants/goal/run_tracker.py

```diff
@@ -133,15 +185,15 @@ def end_run(self, exit_code: ExitCode) -> None:
             raise Exception("RunTracker.end_run() called without calling .start()")
 
         duration = time.time() - self._run_start_time
-        self._total_run_time = duration
+        self._run_total_duration = duration
```

Comment:
See ctor where we set this to None in anticipation of resetting it here, but for some reason used a different name when we actually did so...